cmd/realm/interfaces.go

```diff
@@ -78,6 +80,7 @@ Non-draft interfaces should be removed manually or by your system administrator.
 	Example: `  astartectl realm-management interfaces delete com.my.Interface`,
 	Args:    cobra.ExactArgs(1),
 	RunE:    interfacesDeleteF,
+	Aliases: []string{"remove", "rm", "del"},
```

Comment:
Here I would say the correct verb here is delete (since you're deleting the interface from the system), so I would leave only `del` as alias